Objects/complexobject.c

```diff
@@ -174,12 +174,7 @@ c_powi(Py_complex x, long n)
 {
     Py_complex cn;
 
-    if (n > 100 || n < -100) {
```

Comment:
This branch is now redundant: the bounds check is performed in the code that calls `c_powi`.